governance/executive-council.md

```diff
@@ -0,0 +1,262 @@
+# pyOpenSci Executive Council
+
+The pyOpenSci Executive Council (EC) has four primary functions:
+
+1. **Guides the Organization**: The Executive Council defines and directs pyOpenSci’s high-level [mission, vision, and values](mission-values) and sets its strategic direction.
+
+2. **Advocates for pyOpenSci**: The Executive Council serves as an advocate and ambassador for the organization, leveraging members’ networks to enhance its reputation and support fundraising efforts.
+
+3. **Provides Strategic Insight**: The Executive Council offers perspective and guidance on major strategic decisions to ensure the organization’s long-term success.
+
+4. **Holds the [Executive Director](executive-director) accountable**: The Executive Council holds the Executive Director accountable while also providing high-level strategic and financial guidance and oversight.
+
+The Executive Director is currently a voting member of the Executive Council for items unrelated to their salary and accountability.
+
+The Executive Council holds broad expertise in building and running a scientific / data science nonprofit organization. It is complemented by the [Advisory Council](advisory-council), which offers domain-specific expertise, guidance, and input.
+
+## How the Executive Council works
+
+The pyOpenSci Executive Council operates through open discussions, collaborative
+decision-making, and a commitment to transparency. By leveraging diverse
+perspectives, the Council ensures that its guidance aligns with the organization’s
+[mission, vision, and values](mission-values) while meeting the needs of the
+growing pyOpenSci community.
+
+The Council employs flexible and adaptive practices, such as the **lazy consensus**
+approach, to ensure decisions are made efficiently while fostering thoughtful
```

Comment:
'Lazy consensus' is a tool that we do use around certain types of decisions. But I think having it in this high level section suggests it's our standard approach, which is not true for things like, e.g. budget. So, I would recommend we move this somewhere else, and leave this sentence just as 

'...adaptive practices to ensure decisions are made..."

Reply:
ok removed that part. and we can include it below.